CHANGES.rst

```diff
@@ -59,6 +59,11 @@ New Features
 
 - ``astropy.wcs``
 
+  - wcslib was updated to v5.14 [#4579]
```

Comment:
Not sure what  happened there perhaps it was in an old Changes.rst and wasn't removed when I merged? (there was some conflicts so I did it locally first to fix them)